c2corg_ui/static/js/outingediting.js

```diff
@@ -96,7 +95,7 @@ app.OutingEditingController.prototype.successRead = function(response) {
   // check if user has right to edit -> the user is one of the associated users
   if (this.auth.hasEditRights(outing['associations']['users'])) {
     outing = this.formatOuting_(outing);
-    this.differentDates = app.utils.areDifferentDates(outing['date_start'], outing['date_end']);
+    this.differentDates = (window.moment(outing['date_start']).diff(outing['date_end']) === 0) ? false : true;
```

Comment:
It's a common way in javascript to make sure something is actually a boolean.
Of course ``!!false`` will evaluate to ``false``.
But it also means that ``!!''`` (empty string) or ``!!0`` will evaluate to ``false``.

What does ``window.moment(outing['date_start']).diff(outing['date_end'])`` return? If it is already a boolean, you don't need the ``!!``. Else it depends on the rest of your code, especially of the type of ``this.differentDates``. Note sure Closure compiler will be happy if ``this.differentDates`` is a boolean and you set it to a number or a string.

The thing is to avoid having expression like ``condition ? true : false`` or ``condition ? false : true`` which are a bit of a non-sense since ``condition`` is already a boolean.